test/bin/manage_composer_config.sh

```diff
@@ -0,0 +1,92 @@
+#!/bin/bash
+#
+# A script for managing osbuild-composer.
+
+set -euo pipefail
+
+SCRIPTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+# shellcheck source=test/bin/common.sh
+source "${SCRIPTDIR}/common.sh"
+
+usage() {
+    cat - <<EOF
+${BASH_SOURCE[0]} (create|cleanup|create-workers [num_workers])
+
+  -h           Show this help.
+
+create: Set up system for building images, start webserver.
+
+cleanup: Cancel any running builds, delete failed
+         and completed builds, and remove package 
+         sources other than the defaults.
+
+create-workers: Create multiple osbuild workers for 
+                building in parallel.
+    [num_workers]: Number of workers. If unspecified,
+                   The optimal number of workers will
+                   be determined based on the number
+                   of CPU cores.
+
+EOF
+}
+
+action_create() {
+    if [ $# -ne 0 ]; then
+        usage
+        exit 1
+    fi
+
+    "${ROOTDIR}/scripts/image-builder/configure.sh"
+    
+    "${TESTDIR}/bin/manage_webserver.sh" "start"
+}
+
+action_create-workers() {
+    # If no number is given, determine the optimal number of workers
+    if [ $# -eq 0 ]; then
+        CPU_CORES="$(grep -c ^processor /proc/cpuinfo)"
+        MAX_WORKERS=$(find "${ROOTDIR}/test/image-blueprints" -name \*.toml | wc -l)
+        CUR_WORKERS="$( [ "${CPU_CORES}" -lt  $(( MAX_WORKERS * 2 )) ] && echo $(( CPU_CORES / 2 )) || echo "${MAX_WORKERS}" )"
```

Comment:
Please, convert these var names to lower case and use `local -r var_name` declaration

Reply:
Done